Angeline Dai

JiunHueiDai opened the pull request.

JiunHueiDai committed:
Finished HW2 functionalities

JiunHueiDai committed:
Finished HW3 functionalities

JiunHueiDai committed:
Created initial main java file

JiunHueiDai committed:
User can play until they quit the games

JiunHueiDai committed:
Working on vowels matching for starting with vowel word

JiunHueiDai committed:
Finished rest of functionaities

Need to refactor the code

JiunHueiDai committed:
Refactored the code to be more readable

JiunHueiDai committed:
Finished collection statistics

JiunHueiDai committed:
Finished tic tac toe

JiunHueiDai committed:
First draft of HW7

JiunHueiDai committed:
Finished the gradebook

JiunHueiDai committed:
Refactored code and also added try catch example

danawen commented:
Hi Angeline, I graded your Tic Tac Toe assignment. I took off:
- 5 points because your game let me place pieces in spaces that were already occupied
- 5 points because the game does not inform the user in the event of a tie/draw

JiunHueiDai committed:
Fixed two issues

    5 points because your game let me place pieces in spaces that were already occupied
    5 points because the game does not inform the user in the event of a tie/draw

danawen commented:
> Hi Angeline, I graded your Tic Tac Toe assignment. I took off:
> 
> * 5 points because your game let me place pieces in spaces that were already occupied
> * 5 points because the game does not inform the user in the event of a tie/draw

Re-graded. Thanks!

JiunHueiDai committed:
Has issue when get Age

JiunHueiDai committed:
Finished hw without comments yet

JiunHueiDai committed:
Added comments

JiunHueiDai committed:
Need to work on the Main method in the GarageManager file

JiunHueiDai committed:
Finished Parking Garage with functionalities and comments

JiunHueiDai committed:
Finished car lots program with required hw information

JiunHueiDai committed:
Refactored the code and added more comments in the main program file

JiunHueiDai committed:
Finished rentable hw with comments and requirements

JiunHueiDai committed:
Finished

JiunHueiDai committed:
Finished up the leftover questions

danawen commented:
Hi Angeline, I graded your Rentable assignment. It looks like you're using the value for the Condo weeklyRate and Tool hourlyRate as a daily rate value. You'll need to calculate the daily rate for those two object types based on the weekly and hourly rates, respectively. If you fix that and would like to resubmit, let me know. I'd be happy to re-grade it. Thanks!

JiunHueiDai committed:
Misread the question and now fixed it to calculate daily rate instead

JiunHueiDai committed:
Finished the game but need to refactor the code to make every function has only one single job.

JiunHueiDai committed:
Fixed bug for not being able to replace same charactor in a string

JiunHueiDai committed:
Rename a class, refactored the class, and added comments

JiunHueiDai committed:
Started working on the final project with db

JiunHueiDai committed:
Added all the functionalities but need to test it following by the steps in the video

danawen commented:
Hi @JiunHueiDai - For your Word Guess Game project, I couldn't seem to get the game to work properly. Every time I enter an incorrect character, the game ends.

JiunHueiDai committed:
Fixed bugs

In While loop, if uses "return", it will jump all the way to parent method call.

danawen commented:
> Hi @JiunHueiDai - For your Word Guess Game project, I couldn't seem to get the game to work properly. Every time I enter an incorrect character, the game ends.

@JiunHueiDai Thanks for fixing that! I checked it again and ended up taking off 5 pts because the program doesn't keep track of the incorrect letters guessed (step 6 in the requirements list).

JiunHueiDai committed:
Need to add comments to all methods and separate util to another file

JiunHueiDai committed:
Finished all the comments and bugs and also moved all the common methods to Untils file

JiunHueiDai committed:
Collected user guessed words and missed
